torch/utils/data/dataloader.py

```diff
@@ -154,6 +156,7 @@ class DataLoader(Generic[T_co]):
     pin_memory: bool
     drop_last: bool
     timeout: float
+    device: str
```

Comment:
As device here applied to pin_memory only, please call it `pin_memory_device`

Reply:
will fix it.